src/main/java/core/basesyntax/db/StorageImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl implements Storage {
+    private Map<String, Integer> fruitMap;
+
+    public StorageImpl() {
+        fruitMap = new HashMap<>();
+    }
+
+    @Override
+    public void add(String fruit, int quantity) {
+        fruitMap.put(fruit, quantity);
+    }
+
+    @Override
+    public void change(String fruit, int newQuantity) {
+        fruitMap.replace(fruit, newQuantity);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return fruitMap.get(fruit);
+    }
+
+    @Override
+    public Map<String, Integer> getAll() {
+        return new HashMap<>(fruitMap);
+    }
+}
```

Comment:
According to the checklist, the `Storage` should not have an implementation class. Instead, the `Storage` should be a simple class with a static field. Please refactor the `Storage` to meet this requirement.